functions/src/common.ts

```diff
@@ -25,14 +25,27 @@ export function checkRequest<A>(type: Runtype<A>, data: any) {
 }
 
 /** Return the authenticated user's id or fail if they are not authenticated. */
-export function checkAuth(context: https.CallableContext) {
+export function checkAuth(
+  context: https.CallableContext,
+  checkEmailVerification = false
+) {
   const uid = context.auth?.uid
+
   if (!uid) {
     throw fail(
       "unauthenticated",
       "Caller must be signed in to publish testimony"
     )
   }
+
+  if (checkEmailVerification && process.env.NODE_ENV !== "test") {
```

Comment:
We can use this condition to skip verification checks when running on emulators (also used in triggerPubSubFunction):

```
  if (checkEmailVerification && process.env.FUNCTIONS_EMULATOR !== "true") {
```

Reply:
That's great! thanks 😀